pkg/cmd/server/start/start_master.go

```diff
@@ -446,6 +446,7 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 
 	// Must start policy caching immediately
 	oc.Informers.StartCore(utilwait.NeverStop)
+	oc.Informers.Start(utilwait.NeverStop)
```

Comment:
Ouch... Sorry. I didn't find the place where this was started and I couldn't see  `oc.Informers` started.
Need to investigate, sorry.